lib/licenses.nix

```diff
@@ -55,6 +55,12 @@ in mkLicense lset) ({
     fullName = "GNU Affero General Public License v3.0 or later";
   };
 
+  aladdin = {
```

Comment:
Maybe it's better to name it `afpl` as the Wikipedia [page](https://en.wikipedia.org/wiki/Aladdin_Free_Public_License) suggests?

Reply:
I thought about that but https://spdx.org/licenses/ shows that the SPDX identifier as "Aladdin" which made more sense to match.